spec/ddtrace/benchmark/support/benchmark_helper.rb

```diff
@@ -39,8 +39,9 @@
     @type = e.description
 
     STDERR.puts "Test:#{e.metadata[:example_group][:full_description]} #{e.description}"
+  end
 
-    # Warm up
+  def warm_up
```

Comment:
Warm up was previously being done on a `before` RSpec block, which could run at any point of the setup phase, which can cause issues when benchmark setups are required to run before the warming up starts.

We now moved `warm_up` to inside the test run block, which ensures that it runs after all setup is done.